### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getMetaBoxesPerLocation' of null

### ISSUE THREAD

**Issue by nrherron92** [gocodebox/lifterlms#1617]
### Reproduction Steps

+ Install iThemes Security
+ Open the devoper tools for the console log
+ Click into courses (or the reporting screen)

### Expected Behavior

+ Everything loads fine there's not really a front end issue, this was just found while looking into a speed issue but had no real consequence that I could see. Though, it could maybe cause issues with Classic Editor or that Tinymce editor issue we saw before? 

+ This could totally be an iThemes thing but I know we have a good relationship with them so maybe this is something we should hand in house or if we have a back channel? 

### Error Messages / Logs

+ Include any relevant error messages or log files
```
<!-- Paste error logs / backtraces below this line -->
llms-metaboxes.js:1263 Uncaught TypeError: Cannot read property 'getMetaBoxesPerLocation' of null
    at llms-metaboxes.js:1263
    at llms.js:2210
```

### System and Environment Information

<details>
<summary>System Report</summary>

```
Wordpress
-------------------------------------------

Home Url: [removed]
Site Url: [removed]
Login Url: [removed]/wp-login.php
Version: 5.7.1
Debug Mode: Yes
Debug Log: No
Debug Display: Yes
Locale: en_US
Multisite: No
Page For Posts: Not Set
Page On Front: Home (#2457) [[removed]/]
Permalink Structure: /%postname%/
Show On Front: page
Wp Cron: Yes


Settings
-------------------------------------------

Version: 4.21.1
Db Version: 4.21.1
Course Catalog: Course Catalog (#4751) [[removed]/courses/]
Membership Catalog: Membership Catalog (#4752) [[removed]/memberships/]
Student Dashboard: Dashboard (#1491) [[removed]/dashboard/]
Checkout Page: Purchase (#1490) [[removed]/purchase/]
Course Catalog Per Page: 9
Course Catalog Sorting: menu_order
Membership Catalog Per Page: 9
Membership Catalog Sorting: menu_order
Site Membership: Not Set
Courses Endpoint: my-courses
Edit Endpoint: edit-account
Lost Password Endpoint: lost-password
Vouchers Endpoint: redeem-voucher
Autogenerate Username: yes
Password Strength Meter: yes
Minimum Password Strength: strong
Terms Required: no
Terms Page: Not Set
Checkout Names: required
Checkout Address: required
Checkout Phone: optional
Checkout Email Confirmation: yes
Open Registration: no
Registration Names: required
Registration Address: optional
Registration Phone: hidden
Registration Voucher: optional
Registration Email Confirmation: no
Account Names: required
Account Address: required
Account Phone: optional
Account Email Confirmation: yes
Confirmation Endpoint: confirm-payment
Force Ssl Checkout: no
Country: US
Currency: USD
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Decimals: 2
Trim Zero Decimals: no
Recurring Payments: yes
Email From Address: [removed]
Email From Name: [removed]
Email Footer Text: 
Email Header Image: 
Cert Bg Width: 800
Cert Bg Height: 616
Cert Legacy Compat: no


Constants
-------------------------------------------

LLMS_REMOVE_ALL_DATA: undefined
LLMS_REST_DISABLE: undefined
LLMS_SITE_FEATURE_RECURRING_PAYMENTS: undefined
LLMS_SITE_IS_CLONE: undefined


Gateways
-------------------------------------------

Manual: Disabled
Manual Logging: 
Manual Order: 1


Server
-------------------------------------------

Mysql Version: 5.7.25
Php Curl: Yes
Php Default Timezone: UTC
Php Fsockopen: Yes
Php Max Input Vars: 1000
Php Max Upload Size: 8 MB
Php Memory Limit: 256M
Php Post Max Size: 8M
Php Soap: Yes
Php Suhosin: No
Php Time Limt: 30
Php Version: 7.3.1
Software: Apache
Wp Memory Limit: 40M


Browser
-------------------------------------------

HTTP USER AGENT: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36


Theme
-------------------------------------------

Name: Astra
Version: 3.3.3
Themeuri: https://wpastra.com/
Authoruri: https://wpastra.com/about/
Template: 
Child Theme: No
Llms Support: Yes


Plugins
-------------------------------------------

iThemes Security: 7.9.1
LifterLMS: 4.21.1


Integrations
-------------------------------------------

BbPress: No
BuddyPress: No


Template Overrides
-------------------------------------------




```

</details>

This issue has be recreated:
+ [X] Locally
+ [X] On a staging site
+ [X] On a production website
+ [X] With only LifterLMS and a default theme

**Comment by thomasplevy** [comment]
@nrherron92 fix incoming, this is a conflict we can (and should) easily fix.

**Comment by thomasplevy** [comment]
This was resolved in 4.21.2 (wasn't properly linked)
